gulpfile.babel.js

```diff
@@ -93,10 +98,80 @@ gulp.task('default', ['clean'], cb =>
     'lint',
     'vulcanize',
     'scripts',
+    'generate-service-worker',
     cb
   )
 );
 
+// Copy over the scripts that are used in importScripts as part of the generate-service-worker task.
+gulp.task('copy-sw-scripts', () => {
```

Comment:
You don't need to do this: using `runtimeCaching` will take care of inlining the `sw-toolbox` code.